src/dom.ts

```diff
@@ -140,3 +208,78 @@ export function remove(node: Node) {
 		node.parentNode.removeChild(node);
 	}
 }
+
+/**
+ * Removes all instances of one ore more CSS class names from an Element
+ *
+ * @param element The Element from which to remove CSS classes
+ * @param classes An array of string CSS classes to remove from the Element
+ *
+ * @example
+ * dom.removeClass(document.body, 'loading');
+ *
+ * @example
+ * dom.removeClas(document.body, 'loading', 'pending');
+ */
+export function removeClass(element: Element, ...classes: string[]): void {
+	let targetElement = <any> element;
+	if (!targetElement) {
+		return;
+	}
+	if (targetElement.classList) {
+		let classList: any = targetElement.classList;
+		classList.remove.apply(classList, classes);
+	}
+	else {
+		let oldClasses: string[] = targetElement.className.split(/\s+/);
+		let length = oldClasses.length;
+		for (let className in classes) {
+			validateClass(className);
+			
+			let index = oldClasses.indexOf(className);
+
+			while (index !== -1) {
+				oldClasses.splice(index, 1);
+				index = oldClasses.indexOf(className);
+			}
+		}
+		if (oldClasses.length < length) {
+			targetElement.className = oldClasses.join(' ');
+		}
+	}
+}
+
+/**
+ * Toggles the presence of a CSS class name on an Element. An optional
+ * second parameter can be used to force class addition or removal.
+ *
+ * @param element The Element to add or remove classes to or from
+ * @param className The CSS class name add or remove
+ * @param force Forces either class addition if true or class removal if flase
+ *
+ * @example
+ * dom.toggleClass(button, 'active');
+ *
+ * @example
+ * dom.toggleClass(button, 'active', determineState());
+ */
+export function toggleClass(element: Element, className: string, force?: boolean): boolean {
```

Comment:
It may actually be possible to get away with a default parameter here to simplify logic in the function.  e.g. `force: boolean = containsClass(element, className)`